src/sonic-yang-models/yang-models/sonic-flex_counter.yang

```diff
@@ -159,6 +159,11 @@ module sonic-flex_counter {
                 leaf FLEX_COUNTER_DELAY_STATUS {
                     type flex_delay_status;
                 }
+                leaf POLL_INTERVAL {
+                    type uint32 {
+                        range 1..4294967295;
```

Comment:
I think POLL_INTERVAL range is limited by the CLI. For example - https://github.com/Azure/sonic-utilities/blob/master/counterpoll/main.py#L94.
Do you think we can have the same range here for consistency or you want to relax the constraint of the CLI command?
I am fine with not limiting the upper bound of the polling interval however the lower bound is useful. Setting the counters polling interval to < 100ms will produce a huge load on the CPU.

Reply:
I don't want to relax the constraint of the CLI command, I just couldn't find info about the correct range, so I added a range for uint32 type.
If I understand correctly, are there different ranges for different containers?
For example, for ACL we have range: 1000..30000 - https://github.com/Azure/sonic-utilities/blob/master/counterpoll/main.py#L254
Is this correctly?

Reply:
Yes, it is correct